src/store/chat/utils.ts

```diff
@@ -0,0 +1,24 @@
+export enum JoinRoomApiErrorCode {
+  ROOM_NOT_FOUND = 'ROOM_NOT_FOUND',
+  ACCESS_TOKEN_REQUIRED = 'ACCESS_TOKEN_REQUIRED',
+  GENERAL_ERROR = 'GENERAL_ERROR',
+}
+
+const ERROR_DIALOG_CONTENT = {
+  [JoinRoomApiErrorCode.ROOM_NOT_FOUND]: {
+    header: 'Conversation Not Found',
+    body: 'This conversation does not exist.',
+  },
```

Comment:
Isn't the API supposed to create a room, if it does not exist? How can we get this error code from the API? 🤔